WinUIGallery/Controls/ControlExample.xaml

```diff
@@ -157,7 +157,7 @@
                 HorizontalContentAlignment="Stretch"
                 Background="{ThemeResource CardBackgroundFillColorSecondaryBrush}"
                 CornerRadius="0,0,8,8"
-                Padding="16,0">
+                Padding="0">
```

Comment:
Do we need this if the padding is 0? Or is there a default value we are overriding?

Reply:
Nevermind, we are overriding :)